tools/checked-c-convert/ProgramInfo.h

```diff
@@ -33,80 +33,123 @@
 #include "utils.h"
 #include "PersistentSourceLoc.h"
 
-// A helper class used to track global information in the program. 
-class GlobalSymbol {
+typedef std::set<uint32_t> CVars;
```

Comment:
It'd be helpful to add a comment stating that set actually keeps its contents sorted.    This might be obvious to an expert C++ developer - it will not be obvious to developers less familiar with C.